legion/legion/utils.py

```diff
@@ -573,7 +572,7 @@ def get_list_of_requirements():
     """
     file_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'data', 'Pipfile.lock')
     if not os.path.exists(file_path):
-        raise Exception('File with requirements ({}) is not exists', file_path)
+        raise Exception('File with requirements ({}) is not exists', file_path)  # pylint: disable=W0715
```

Comment:
Why did not you fix this?

Reply:
Not sure I understand what is the fix here.
To use `.format(file_path)` instead of passing `file_path` as argument?